com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -539,7 +539,7 @@ void UpdateSensors()
         /// Visual observations are implicitly added from the cameras attached to
         /// the Agent.
         /// </remarks>
-        public virtual void CollectObservations()
+        public virtual void CollectObservations(VectorSensor sensor)
```

Comment:
Please update the docstring for this.

Reply:
Let me know what you think

Reply:
Sounds good. I'm not sure about the syntax on cref, but we can sort that out when we start building the package docs.